pkg/kubelet/nodeshutdown/nodeshutdown_manager_windows.go

```diff
@@ -51,12 +51,6 @@ const (
 	shutdownOrderStringValue = "PreshutdownOrder"
 )
 
-const (
-	nodeShutdownNotAdmittedReason  = "NodeShutdown"
```

Comment:
As you renamed it to NodeShutdownNotAdmittedReason, the line 123 here needs to be updated as well.

Reply:
https://github.com/kubernetes/kubernetes/blob/99140b2ec2b2c7107b2370d6c3e78cf8883f1652/pkg/kubelet/nodeshutdown/nodeshutdown_manager_windows.go#L117

Reply:
Fixed